spec/models/manageiq/providers/hawkular/middleware_manager/refresher_spec.rb

```diff
@@ -30,19 +30,20 @@
 
     # check whether the server was associated with the vm
     server = @ems_hawkular.middleware_servers.first
-    expect(server.lives_on_id).to eql(@vm.id)
-    expect(server.lives_on_type).to eql(@vm.type)
+    # TODO: Restore these expectations
```

Comment:
They should appear if the hawkular-services are deployed on a virtual machine managed by RHEV provider and this provider is also added to MiQ. The cross-linking should just happen automatically. If I recall correctly the https://hawkular.torii.gva.redhat.com is running on RHEV and both the hawkular and RHEV can be added as providers to MiQ (juca can give you the access). Although, it will probably not work for the containerized hawular-services, because we no longer have the 

```bash
sudo echo $HOSTNAME > /etc/machine-id
```

..in the start script, because the docker image must run without the root on openshift.

@jpkrohling As the author, could you pls check that I am not lying here :] Also, do you have any idea how to workaround the ^ for containers? Perhaps we can change the permissions of `/etc/machine-id` so that it's writable by non-root guys, right? That would work. If you are not against it I am willing to add it back.